index.bs

```diff
@@ -652,6 +651,8 @@ Each {{XRSession}} has an <dfn for=XRSession>animation frame</dfn>, which is an
 
 Each {{XRSession}} has a <dfn for=XRSession>set of granted features</dfn>, which is a [=/set=] of {{DOMString}}s corresponding to the [=feature descriptors=] that have been granted to the {{XRSession}}.
 
+The <dfn attribute for="XRSession">enabledFeatures</dfn> attribute returns the features in the [=XRSession/set of granted features=] as an array of {{DOMString}}s.
```

Comment:
hmm, i think we probably want it to be a method? 

If it's a frozenarray we need to spec whether the array is cached and what happens when there's a change. Either way we need algorithm steps for this.

Reply:
My intention here (based on what it looked like the discussion in the issue was), was that it would not be a SameObject so just generated by demand by adding all the elements of the set to a list and returning that. I can write an algorithm for that, but it seemed fairly trivial/just one step. Given the other usages of FrozenArray's within the spec (including as attribute's), I didn't think it needed to be a method, can you explain what the benefit of turning it into a method would be?

(I also don't feel strongly that it needs to be a FrozenArray, if we feel that a setlike is better; but in the issue discussion it sounded like there were some benefits to a FrozenArray).

Reply:
Yeah I guess as a non-SameObject attribute it's fine, we should probably just explicitly mention that in the spec text as well.

Reply:
As an algorithm, or perhaps by making this line:

`...returns the features in the [=XRSession/set of granted features=] as a new array of {{DOMString}}s`

?

Reply:
Yeah.

Reply:
Done.